frontend_server/lib/server.dart

```diff
@@ -298,9 +277,9 @@ Future<int> starter(
           boundaryKey = string.substring(RECOMPILE_INSTRUCTION_SPACE.length);
           state = _State.RECOMPILE_LIST;
         } else if (string == 'accept') {
-          compiler.acceptLastDelta();
+          // NOP
```

Comment:
Perhaps add a TODO explaining the change to track removal of these commands from flutter/flutter too?

Reply:
We might need to restore this. I am iterating on this with Peter.